package-lock.json

```diff
@@ -1,8566 +0,0 @@
-{
```

Comment:
Your work should be done in your folder only.
Please remove **changes** of this file from your pull request.

Reply:
I have already deleted package-lock.json from my pull request. I did it in "files changed"  and deleted file. Its ok for now?

Reply:
No, I'm afraid, that the changes of this file are still in your pr. So, if I merge your pr, these changes will overwrite package-lock.json of whole project.
Here I've added some description of the problem https://t.me/c/1602765599/177

Reply:
I checked your description.  I can t see changes in package-lock.json because i deleted file before. Maybe I should close my pull request and create it again?

<img width="1393" alt="Снимок экрана 2023-10-23 в 13 28 41" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/89414082/a32e7c22-9866-48f1-9dd3-456ff8b74490">

Reply:
You should not DELETE the FILE. You should REMOVE its CHANGES from this pull request. That means, that you should REVERT the FILE to the INITIAL state - like in the main branch https://github.com/MastersAcademy/fe-course-basic-2023/blob/main/package-lock.json
So, if I merge your pr now, your changes (DELETION of the file) will gor to the main branch, and the file will be deleted there.
Ask your colleagues in the[ Flood chat,](https://t.me/+eQspq3eMIeNkMGVi) I believe somebody help you.